packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -300,6 +300,7 @@ async function initializeRoute() {
 	}
 
 	const isAlreadyInitialized =
+		route.params.name === initializedWorkflowId.value &&
 		initializedWorkflowId.value &&
 		[NEW_WORKFLOW_ID, workflowId.value].includes(initializedWorkflowId.value);
 
```

Comment:
Is there a test we can add to check this change?

Reply:
As there are no tests right now, we can merge this but look at testing as the next steps.